### PR TITLE
Tests & refactor package_helper#guess_code_class

### PR DESCRIPTION
**Pull request by Ana06**
I added rspec tests for `package_helper#guess_code_class(filename)`. Property tests were used, as they are really convenient when testing regular expressions. :bowtie: 

Also, I refactored `package_helper#guess_code_class` after adding tests.

### CODE REVIEW

**Comment by Ana06** [comment]
[(1) P13: app/helpers/webui/package_helper.rb](https://trello.com/c/9TsNZOO4/396-1-p13-app-helpers-webui-package-helper-rb)

File: src/api/spec/helpers/webui/package_helper_spec.rb [hunk 1]
```diff
@@ -68,7 +69,246 @@
...
+
+    context "css" do
+      it "with css extension" do
+        property_of {
+          sized(range(1, 195)){ string(/[\w+\-:\.]/) } + '.css'
+        }.check(3) { |filename|
+          expect(guess_code_class(filename)).to eq('css')
+        }
+      end
+    end
```
Comment (bgeuken) [inline src/api/spec/helpers/webui/package_helper_spec.rb]: With that much duplication I would consider using a shared example here

Comment (Ana06) [inline src/api/spec/helpers/webui/package_helper_spec.rb]: yes, great idea! :+1:

**Comment by bgeuken** [comment]
:+1:

**Pull request merged**